Rakefile

```diff
@@ -11,7 +11,7 @@ task :console do
   if File.exist? config_file
     config = YAML.load File.read(config_file)
 
-    AWS.config(config['aws'])
+    Aws.config = config['aws']
```

Comment:
Well, changing the Shoryuken logger isn't aws config so you wouldn't put it in there anyway. You can call ```Shoryuken::Logging.logger = Logger.new(STDOUT)``` at any time to change the logger, but it should be logging to STDOUT anyway: see https://github.com/musicglue/shoryuken/blob/aws-sdk-core/lib/shoryuken/logging.rb#L28

I'm running Shoryuken with Rails, so I just dump any extra configuration code in ```config/initializers/shoryuken.rb```. That takes care of making sure it's executed for rake tasks, console sessions, etc. Is your situation different?

On a related note, there's a good argument for extracting all the config file reading stuff (e.g. https://github.com/musicglue/shoryuken/blob/aws-sdk-core/lib/shoryuken/cli.rb#L232) from ```Shoryuken::CLI``` and putting it somewhere else. Perhaps then it could be automatically loaded the first time you do anything with a Shoryuken object. You probably wouldn't need a ```:console``` task then. As it stands, this console rake task won't work if you have any non-Aws compatible keys in your yaml because it doesn't know to filter them out.